### PR TITLE
#1206: Add new styling for highlighted operations

### PR DESCRIPTION
**Pull request by batpad**
Refs #1206 -

@imohkay just opening this PR for visibility. I will work through the items you've flagged for me on at https://github.com/IFRCGo/go-frontend/issues/1206#issuecomment-653913419

cc @frozenhelium for visibility

### CODE REVIEW

File: src/root/components/highlighted-operations/operation-card.js [hunk 1]
```diff
@@ -14,13 +14,18 @@ const OperationCard = ({operationId, operationName, emergencyDeployments, appeal
...
-        <div className="card_box card_box_left">
-          <h2 className='card__title'>{ operationName.length > 30 ? operationName.slice(0, 30) + '...' : operationName }</h2>
-          <small className='last_updated'>
-            <Translate stringId='operationCardLastUpdated'/>
-            &nbsp;
-            {formatDate(lastUpdate)}
-          </small>
+        <div className="card_box card_box_left row">
+          <div className='card__title__wrap'>
+            <h2 className='card__title'>{ operationName.length > 30 ? operationName.slice(0, 30) + '...' : operationName }</h2>
```
Comment (frozenhelium) [inline src/root/components/highlighted-operations/operation-card.js:19]: Not that these are change in this PR but just wanted to point out one thing here:
We should use CSS to properly truncate the text, 
```css
.card__title {
  white-space: nowrap;
  overflow: hidden;
  text-overflow: ellipsis;
}
```
for more info: https://developer.mozilla.org/en-US/docs/Web/CSS/text-overflow

Comment (imohkay) [reply src/root/components/highlighted-operations/operation-card.js:19]: Okay. Is that a requirement that the title is only one line? Then we can do the needful @frozenhelium 
I suppose we can also make a mixin and reuse as needed...

Comment (batpad) [reply src/root/components/highlighted-operations/operation-card.js:28]: @imohkay right now the javascript is doing the truncation to 30 characters in the `.slice(...)` operation - I think the idea here would be to not truncate in Javascript and have CSS handle the display with ellipsis for overflow. Happy to help with the JS end of this if you can implement the text-overflow: ellipsis bit to look neat, etc.

**Comment by batpad** [comment]
@frozenhelium can I ask you to please review my updates to the branch? The implementation of the Follow / Unfollow logic seemed to get a bit strange - let me know if it helps to talk through, I feel like my React skills are getting rusty.

File: src/root/components/highlighted-operations/operation-card.js (hunk 1 above)
Comment (batpad) [reply src/root/components/highlighted-operations/operation-card.js:28]: Per voice with @imohkay - let's ticket and do this separately.

**Comment by batpad** [comment]
This has all been combined into #1369 - closing here.

**Pull request closed without merging**